AndroidAnnotations/androidannotations-ormlite/ormlite/src/main/java/org/androidannotations/ormlite/handler/OrmLiteDaoHandler.java

```diff
@@ -96,7 +116,7 @@ public void process(Element element, EComponentHolder holder) {
 		catchBlock.body() //
 				.staticInvoke(getClasses().LOG, "e") //
 				.arg(holder.getGeneratedClass().name()) //
-				.arg("Could not create DAO " + fieldName) //
+				.arg("Could not create DAO") //
```

Comment:
Maybe you could pass the `element` as an argument of the method `wrapInjection` so that we can improve the log.